pkg/broker/broker.go

```diff
@@ -467,9 +487,34 @@ func mergeCredentials(
 	return provExtCreds.Credentials
 }
 
-// Unbind - unbind a services previous binding NOTE: not implemented.
-func (a AnsibleBroker) Unbind(instanceUUID uuid.UUID, bindingUUID uuid.UUID) error {
-	return notImplemented
+// Unbind - unbind a services previous binding
+func (a AnsibleBroker) Unbind(instanceUUID uuid.UUID, bindingUUID uuid.UUID) (*UnbindResponse, error) {
+	if _, err := a.dao.GetBindInstance(bindingUUID.String()); err != nil {
+		return nil, ErrorNotFound
+	}
+
+	serviceInstance, err := a.GetServiceInstance(instanceUUID)
+	if err != nil {
+		a.log.Debugf("Service instance with id %s does not exist", instanceUUID.String())
+	}
+
+	err = apb.Unbind(serviceInstance, a.clusterConfig, a.log)
+	if err != nil {
+		return nil, err
+	}
+
+	err = a.dao.DeleteBindInstance(bindingUUID.String())
+	if err != nil {
+		return nil, err
+	}
+
+	serviceInstance.RemoveBinding(bindingUUID)
+	err = a.dao.SetServiceInstance(instanceUUID.String(), serviceInstance)
+	if err != nil {
+		return nil, err
+	}
+
+	return &UnbindResponse{Operation: "successful"}, nil
```

Comment:
This is not spec compliant. The spec states:
>Body
>
>All response bodies MUST be a valid JSON Object ({}). This is for future compatibility; it will be easier to add fields in the future if JSON is expected rather than to support the cases when a JSON body might or might not be returned.
>
>For a success response, the expected response body is {}.